src/time.rs

```diff
@@ -106,6 +89,7 @@ impl<'a> FromDer<'a> for Time {
     const TYPE_ID: DerTypeId = DerTypeId::Time;
 }
 
+/*
```

Comment:
Why not remove this outright?

Reply:
I wasn't sure yet if anything needed it. :)